test/integration/targets/nxos_command/tests/cli/not_comparison_operator.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- debug: msg="START cli/not_comparison_operator.yaml"
+
+- name: test 'not' keyword in wait_for
+  nxos_command:
+    commands:
+      - show version
+      - show ver | i QWERTYQWERTYQWERTY
+    wait_for:
+      - "result[0] not contains QWERTYQWERTYQWERTY"
+      - "result[0] == not QWERTYQWERTYQWERTY"
+      - "result[0] matches not QWERTYQWERTYQWERTY"
+      - "result[1] neq not ''"
+    provider: "{{ cli }}"
```

Comment:
@eakman Can you please add the test in nxos_command/tests/common/ instead? so the test will run against both transports as this will be supported for nxapi also.
Also no need to add `provider`.